ci/gpu/build.sh

```diff
@@ -16,6 +16,7 @@ function hasArg {
 # Set path and build parallel level
 export PATH=/opt/conda/bin:/usr/local/cuda/bin:$PATH
 export PARALLEL_LEVEL=${PARALLEL_LEVEL:-4}
+export CONDA_ARTIFACT_PATH=${WORKSPACE}/ci/artifacts/cugraph/cpu/.conda-bld/
```

Comment:
```suggestion
export CONDA_ARTIFACT_PATH=${WORKSPACE}/ci/artifacts/cuml/cpu/.conda-bld/
```

should this be `cuml`?

Reply:
Yup! Forgot to push that commit, good catch!